### ISSUE TITLE
Neo4j 3.3.5 LOAD CSV throw Neo.DatabaseError.Statement.ExecutionFailed

### ISSUE THREAD

**Issue by JunOkumura** [neo4j/neo4j#11687]
**Neo4j Version:** 3.3.5
**Operating System:** OS X 10.13.4
**Driver:**  Cypher

### Steps to reproduce
1. Access to own Neo4j browser.
2. Run the below cypher.
```
USING PERIODIC COMMIT LOAD CSV WITH HEADERS FROM 'file:///test.csv' AS row CREATE (tx:TxOut { value: row.value, n: toInt(row.n), uuid: row.uuid, created_at: timestamp() }) SET tx.script_pubkey = row.script_pubkey, tx.updated_at = timestamp()
```
### Expect behavior
Import finishes without error.

### Actual behavior
That return the error.
```
Neo.DatabaseError.Statement.ExecutionFailed: Tried to read a field larger than buffer size 2097152. A common cause of this is that a field has an unterminated quote and so will try to seek until the next quote, which ever line it may be on. This should not happen if multi-line fields are disabled, given that the fields contains no new-line characters. This field started at <NEO4J-HOME>/import/test.csv:1
```

using file [here test.zip](https://github.com/neo4j/neo4j/files/1965963/test.zip)

**Comment by SaschaPeukert** [comment]
@JunOkumura Thanks for reporting. I could reproduce it and this seems like a bug to me. I will let you know when it's fixed.

Best regards 
Sascha, Neo4j Cypher Team

**Comment by SaschaPeukert** [comment]
This should now be fixed in `3.3.6`
